src/silx/gui/plot/backends/BackendOpenGL.py

```diff
@@ -907,7 +912,7 @@ def _lineStyleToDashOffsetPattern(
             pattern = ()
         if len(pattern) == 2:
             pattern = pattern * 2
-        return offset, pattern
+        return float(offset), tuple(float(v) for v in pattern)
```

Comment:
Not related to this PR but missing from adding `int` as a accepted type for line style.